index.php

```diff
@@ -28,8 +28,20 @@
 
         <nav class="user-menu">
 
-        <!-- здесь должен быть PHP код для показа имени пользователя -->
-
+        <?php if ($is_auth): ?>
+        <div class="user-menu__logged">
+            <p><?=$user_name;?></p>
+        </div>
+        <?php else: ?>
+        <ul class="user-menu__list">
```

Comment:
```suggestion
            <ul class="user-menu__list">
```
В таких местах можно делать лесенку из кода
....if
........ul
............li